app/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "build-dev": "vue-cli-service build --dest ../api/dist",
```

Comment:
This tells the builder to place the dist folder in the api folder for dev purposes.

Reply:
Well done.